bin/hdfcoinc/pycbc_coinc_statmap

```diff
@@ -19,112 +22,216 @@ class fw(object):
+        # Make a new item if isn't in the hdf file
+        if not name in self.f:
+            self.f.create_dataset(name, data=data, compression="gzip",
+                                  compression_opts=9, shuffle=True,
+                                  maxshape=data.shape)
+        # Else re-size that section and reassign values
+        else:
+            self.f[name].resize(data.shape)
+            self.f[name][:] = data
+
+    def __getitem__(self, *args):
+        return self.f.__getitem__(*args)
+
+    def close(self):
+        self.f.close()
 
 parser = argparse.ArgumentParser()
 # General required options
 parser.add_argument('--version', action='version', 
-         version=pycbc.version.git_verbose_msg)
+                    version=pycbc.version.git_verbose_msg)
 parser.add_argument('--coinc-files', nargs='+', 
-         help='List of coincidence files used to calculate the FAP, FAR, etc.')
+                    help='List of coincidence files used to calculate the '
+                         'FAP, FAR, etc.')
 parser.add_argument('--verbose', action='count')
 parser.add_argument('--cluster-window', type=float, default=10,
-         help='Length of time window in seconds to cluster coinc events, [default=10s]')
+                    help='Length of time window in seconds to cluster coinc '
+                         'events, [default=10s]')
 parser.add_argument('--veto-window', type=float, default=.1,
-         help='Time around each zerolag trigger to window out, [default=.1s]')
+                    help='Time around each zerolag trigger to window out, '
+                         '[default=.1s]')
+parser.add_argument('--veto-window-hierarch', type=float, default=1.0,
+                    help='Time around each trigger to window out for a very '
+                         'louder trigger in the hierarchical removal '
+                         'procedure. [default=1.0s]')
+parser.add_argument('--cont-hier-removal', action='store_true',
+                    default='store_false', 
+                    help='Use this option to do continuous hierarchical '
+                         'removal without any cutoff number of hierarchical '
+                         'removals to stop at. [default=false]')
+parser.add_argument('--max-hier-removal', type=int, default=0,
+                    help='Maximum amount of hierarchical removals to carry '
+                         'out when hierarchical removal is desired. Choose '
+                         '0 for no hierarchical removal. Choose 1 to do at '
+                         'most 1 hierarchical removal, and so on. The code '
+                         'will write the triggers out for whichever comes '
+                         'first: that there are no more hierarchical removals '
+                         'to do, or the max allowed amount of hierarchical '
+                         'removals has been reached. [default=0]')
+parser.add_argument('--store-last-h-rm-background', action='store_true',
```

Comment:
I guess better would be to just fix the plotting code so that it asks which inclusive background you want to plot. This is a fair comment. I think I'll do that.